File: LLM-Translations/GPT-4/codenet/Python/C/s747093551.c

#include <stdio.h>

int main() {
    int N, M, X, Y, i, max_x = -100, min_y = 101;
    scanf("%d %d %d %d", &N, &M, &X, &Y);

    int x[N], y[M];
    for (i = 0; i < N; i++) {
        scanf("%d", &x[i]);
        if (x[i] > max_x) {
            max_x = x[i];
        }
    }
    for (i = 0; i < M; i++) {
        scanf("%d", &y[i]);
        if (y[i] < min_y) {
            min_y = y[i];
        }
    }

    if (max_x < X) {
        max_x = X;
    }
    if (min_y > Y) {
        min_y = Y;
    }

    if (max_x < min_y) {
        printf("No War\n");
    } else {
        printf("War\n");
    }

    return 0;
}

File: LLM-Translations/GPT-4/codenet/Python/Go/s747093551.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var N, M, X, Y int
	fmt.Scan(&N, &M, &X, &Y)

	x := make([]int, N)
	for i := 0; i < N; i++ {
		fmt.Scan(&x[i])
	}

	y := make([]int, M)
	for i := 0; i < M; i++ {
		fmt.Scan(&y[i])
	}

	maxX, minY := math.MinInt64, math.MaxInt64
	for _, v := range x {
		if v > maxX {
			maxX = v
		}
	}
	for _, v := range y {
		if v < minY {
			minY = v
		}
	}

	if max(maxX, X) < min(minY, Y) {
		fmt.Println("No War")
	} else {
		fmt.Println("War")
	}
}

func max(a, b int) int {
	if a > b {
		return a
	}
	return b
}

func min(a, b int) int {
	if a < b {
		return a
	}
	return b
}
// 
